ls1intum/Artemis
# WIP: Kotlin Template

### Checklist

### Motivation and Context
We needed to create multiple Kotlin exercises for the PSE Lecture and encountered a lot of issues when just creating them as Java projects. This creates a template with all the extras that we needed to get it running.

### Description
This PR only adds the extra programming language and the necessary template. This PR has not yet been tested. I'm not quite sure how to run it

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Administration
3. Create a programming exercise with Kotlin as the template
4. The template should perform just like the java template just with Kotlin files instead of Java files
5. Cloning the repository and importing it as a maven project should work out of the box
